salesapp.js

```diff
@@ -2,6 +2,21 @@
 
 var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm'];
 var cookieStores = [];
+var addStoreForm = document.getElementById('add_store_form');
+var tableEl = document.getElementById('cookieTable');
+
+addStoreForm.addEventListener('submit', function(event) {
+  event.preventDefault();
+  var minCustPerHour = parseInt(document.getElementById('min_cust_per_hour').value),
+    maxCustPerHour = parseInt(document.getElementById('max_cust_per_hour').value),
+    avgCookiesPerCust = parseFloat(document.getElementById('avg_cookies_per_hour').value),
+    storeName = document.getElementById('shop_location').value;
+  new CookieStore(minCustPerHour, maxCustPerHour, avgCookiesPerCust, storeName);
+  tableEl.textContent = '';
+  makeTableHeader();
+  renderCookieStores();
+  makeTableFooter();
```

Comment:
It's somewhat inefficient to have to erase and regenerate the *entire* table every time you add a new store. In this instance it works, but consider how much of a headache it might be if you had 10,000 stores to manage? It's far less intensive to simply add one new row at the bottom of your table. If you're concerned about having the footer, then you can simply delete the footer, add your new row, then add the footer back on.